### DISCUSSION TITLE
Cloudflared with HomeAssistant Nabu Casa URL?

### DISCUSSION THREAD

**Discussion by Austin1** [homeassistant-apps/app-cloudflared#407] [Q&A] [answered]
So I got cloudflared running and working, but when I use it with Nabu Casa URL, it doesn't let me into other resources on my network that are in a separate cloudflare tunnel. If I were to use additional_hosts configuration in the cloudflared tunnel, will it translate the URL and work with Nabu Casa or do I just dump nabu casa and use cloudflared only?

**Comment by brenner-tobias** [comment, marked as answer]
Everything that you add to the additional_hosts will be accesable via the Cloudflare tunnel. The nabu casa connection is not aware of these hosts, since they are not really HA related, you just use the Cloudflare tunnel that is hosted in a container on the HA OS. So you have to use the Cloudflare URL for these hosts.
